docs/index.md

```diff
@@ -3,17 +3,17 @@
   title: ""
 ---
 
-# AddressBook Level-3
+# EstateEase
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
```

Comment:
I just realised, but our status icons still point to AB3.

Here's the update one from our README
```
[![CI Status](https://github.com/AY2324S2-CS2103-F09-1/tp/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions) [![codecov](https://codecov.io/gh/AY2324S2-CS2103-F09-1/tp/graph/badge.svg?token=NB86I9H23Z)](https://codecov.io/gh/AY2324S2-CS2103-F09-1/tp)
```

Preview:
[![CI Status](https://github.com/AY2324S2-CS2103-F09-1/tp/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions) [![codecov](https://codecov.io/gh/AY2324S2-CS2103-F09-1/tp/graph/badge.svg?token=NB86I9H23Z)](https://codecov.io/gh/AY2324S2-CS2103-F09-1/tp)

Reply:
Ohh okay I will change this